cmd/serve.go

```diff
@@ -91,7 +109,7 @@ func serve(_ context.Context) {
 		oauth2.NewClientCredentialsHandlerFactory,
 	)
 
-	apiHandler, err := httpsrv.NewAPIHandler(storageEngine)
+	apiHandler, err := httpsrv.NewAPIHandler(storageEngine, auditMiddleware)
```

Comment:
You could instead provide the middleware to echox.Config which applies it to all routes. (Additionally it looks like you need to rebase as this is already being used in identity api)

Reply:
thanks, ill rebase - I dont think we want the middleware applied to all routes? ie do we really want to audit every call to the jwks endpoint?